setup.py

```diff
@@ -25,4 +23,7 @@
         ],
     packages = ['isbn'],
     package_dir = {'isbn': 'isbn'},
-    )
+    autosemver={
+        'bugtracker_url': True,
```

Comment:
Just checked, actually it does not give a false positive :), but does not detect issues on the autosemver parameters (that would be really amazing!)